Ilya/NetflixRoulette/netflix-roulette/src/pages/home/HomePage.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import queryString from "query-string";
+import { RouteComponentProps } from "react-router-dom";
+
+import "./HomePage.scss"
+import Header from 'shared/header/Header';
+import SearchMovie from "shared/searchMovie/SearchMovie";
+import Footer from 'shared/footer/Footer';
+import MovieList from 'shared/movieList/MovieList';
+import MovieSorter, { SortBy } from "shared/movieSorter/MovieSorter";
+
+interface IMovie {
```

Comment:
move to a separate file or file with reusable interfaces. You are already using IMovie in App component